docs/integrations/public-ip.md

```diff
@@ -0,0 +1,25 @@
+# Pipes' Public IP Addresses
+
+## **Overview**
+This document provides a list of public IPs used by Pipes for outbound requests. These IPs are intended for end-users to configure whitelisting, firewall rules, and integrations with external services.
+
+## **Public IPs**
+#### **1. Load Balancer IPs**
+The following IPs are used by Pipes' load balancers. Please ensure these are whitelisted for access to our services:
+
+| **Load Balancer Name**       | **Static IP Address**     | **Ports**     |
+|------------------------------|---------------------------|---------------|
+| Main Service           | 34.86.218.179            | 80, 443       |
+| Database Proxy                    | 34.145.175.31            | 9193          |
```

Comment:
Probably best to be more Pipe-specific? `Steampipe DB`?

Reply:
Makes sense. Updated to `Steampipe Database Proxy`

Reply:
The DB and dashboard IPs are really per hive; we just happen to only have one hive at present.   It may be more accurate to look them up for your individual workspace.  For example, you can get your database host from the Steampipe **query** tab by clicking the **info** button, eg, `myorg-workspace.usea1.db.pipes.turbot.com`.   The dashboard endpoint is the same as the db, replacing the `.db` segment with `.dashboard`, eg  `myorg-workspace.usea1.dashboard.pipes.turbot.com`.

Reply:
I would not call these "load balancer IPs" - that's not really relevant to the end user.   Note that the scope is different - what you call **Application Endpoint** is the pipes.turbot.com console and API endpoint, and it's global for everyone.  **Steampipe Database** and **Powerpipe Dashboard** are for *your workspace* and could vary from one workspace to the next (eventually).

Reply:
@prithage perhaps have a chat with @johnsmyth on this to get the required path for updates?